pkg/sql/distsqlrun/joinreader_test.go

```diff
@@ -268,6 +268,38 @@ func TestJoinReader(t *testing.T) {
 			outputTypes: sqlbase.OneIntCol,
 			expected:    "[['two']]",
 		},
+		{
+			description: "Test left semi lookup join",
+			indexIdx: 1,
+			post: distsqlpb.PostProcessSpec{
+				Projection: true,
+				OutputColumns: []uint32{0, 1},
+			},
+			input: [][]tree.Datum{
+				{tree.NewDInt(tree.DInt(0)),  sqlutils.RowEnglishFn(2)},
+			},
+			lookupCols:  []uint32{0},
+			joinType:    sqlbase.LeftSemiJoin,
+			inputTypes:  []types.T{*types.Int, *types.String},
+			outputTypes: sqlbase.TwoIntCols,
+			expected:    "[[0 'two']]",
```

Comment:
Should correct output be `[[0 'two'] [0 'two'] [0 'two'] [0 'two'] [0 'two'] [0 'two'] [0 'two'] [0 'two'] [0 'two']]` because we have 9 instances of 0 in the table?

Reply:
Yes, if the left side has n tuples, each of which has a match on the right side, there should be n copies of the left side in the output.